File: benchmarks/fib.py
#!/usr/bin/env python3
# Fibonacci benchmark - recursive (CPU-bound)

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

if __name__ == "__main__":
    print(fib(35))

File: benchmarks/fib.c
// Fibonacci benchmark - recursive (CPU-bound)
#include <stdio.h>

int fib(int n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(void) {
    int result = fib(35);
    printf("%d\n", result);
    return 0;
}
